src/test/java/javax/jmdns/test/JmDNSTest.java

```diff
@@ -431,6 +432,7 @@ public boolean waitForReceive() {
     private final static int MPORT = 8053;
 
     @Test
+    @Ignore
```

Comment:
@kaikreuzer Do you happen to remember why it was set to `@Ignored`?

Reply:
No, sorry, I don't. Possibly because none of them were working...